module/VuFindSearch/src/VuFindSearch/Backend/Solr/QueryBuilder.php

```diff
@@ -460,18 +460,17 @@ protected function reduceQueryGroupComponents(AbstractQuery $component)
                 [$this, 'reduceQueryGroupComponents'],
                 $component->getQueries()
             );
-            $searchString = $component->isNegated() ? 'NOT ' : '';
             $reduced = array_filter(
                 $reduced,
                 function ($s) {
                     return '' !== $s;
                 }
             );
-            if ($reduced) {
-                $searchString .= sprintf(
-                    '(%s)',
-                    implode(" {$component->getOperator()} ", $reduced)
-                );
+            $searchString = $reduced
+                ? sprintf('(%s)', implode(" {$component->getOperator()} ", $reduced))
```

Comment:
Since we usually just do concatenation instead of sprintf, I'd favor it here as well. Needs extra parenthesis, but still.